src/internal/packager/git/fetch.go

```diff
@@ -15,35 +15,49 @@ import (
 	goConfig "github.com/go-git/go-git/v5/config"
 )
 
-// fetchTag performs a `git fetch` of _only_ the provided tag.
-func (g *Git) fetchTag(tag string) {
-	message.Debugf("git.fetchTag(%s)", tag)
-
-	refspec := goConfig.RefSpec("refs/tags/" + tag + ":refs/tags/" + tag)
+// fetchRef performs a `git fetch` of _only_ the provided git reference (tag or hash).
+func (g *Git) fetchRef(ref string) error {
+	var err error
+
+	if IsHash(ref) {
+		err = g.fetchHash(ref)
+	} else {
+		err = g.fetchTag(ref)
+	}
 
-	err := g.fetch(g.GitPath, refspec)
+	return err
+}
 
-	if err != nil {
-		message.Fatal(err, "Not a valid tag or unable to fetch")
+// fetchTag performs a `git fetch` of _only_ the provided tag.
+func (g *Git) fetchTag(tag string) error {
+	message.Debugf("git.fetchTag(%s)", tag)
```

Comment:
I thought we decided we weren't adding debug message anymore?

Reply:
I didn't actually change this line but git thinks I did because of other changes above/below this so I didn't catch that. I'll remove it now :)